bevy_lint/src/lints/nursery/camera_modification_in_fixed_update.rs

```diff
@@ -0,0 +1,194 @@
+//! Checks for systems added to the `FixedUpdate` schedule that mutably query entities with a
+//! `Camera` component.
+//!
+//! # Motivation
+//!
+//! Modifying camera components in the `FixedUpdate` schedule can cause jittery or inconsistent
+//! visuals because `FixedUpdate` runs at a fixed timestep mainly for physics and deterministic
+//! logic. The camera should be updated in the last variable timestep before `FixedUpdate` which is
+//! the `Update` schedule.
+//!
+//! # Known Issues
+//!
+//! - The lint only detects systems that explicitly use the `With<Camera>` query filter.
+//!
+//! # Example
+//!
+//! ```rust
+//! use bevy::prelude::*;
+//!
+//! fn move_camera(mut query: Query<&mut Transform, With<Camera>>) {
+//!     // ...
+//! }
+//!
+//! fn main() {
+//!     App::new()
+//!         .add_systems(FixedUpdate, move_camera);
+//! }
+//! ```
+//!
+//! Use instead:
+//!
+//! ```rust
+//! use bevy::prelude::*;
+//!
+//! fn move_camera(mut query: Query<&mut Transform, With<Camera>>) {
+//!     // ...
+//! }
+//!
+//! fn main() {
+//!     App::new()
+//!         .add_systems(Update, move_camera);
+//! }
+//! ```
+
+use clippy_utils::{diagnostics::span_lint_and_help, sym, ty::match_type};
+use rustc_hir::{ExprKind, QPath, def::Res};
+use rustc_lint::{LateContext, LateLintPass};
+use rustc_middle::ty::{Adt, GenericArgKind, TyKind};
+use rustc_span::Symbol;
+
+use crate::{declare_bevy_lint, declare_bevy_lint_pass, utils::hir_parse::MethodCall};
+
+declare_bevy_lint! {
+    pub CAMERA_MODIFICATION_IN_FIXED_UPDATE,
+    super::Nursery,
+    "Camera modification in FixedUpdate schedule",
```

Comment:
```suggestion
    "camera modified in the `FixedUpdate` schedule",
```

(Nit) This makes the description a bit more consistent with the other lints. While we don't have any official guidelines for lint descriptions, I plan on writing something down in the next week or so!